docs/_sass/getnighthawk.scss

```diff
@@ -5445,5 +5445,113 @@ a.globalFooterCard.card-environment, div.globalFooterCard.card-environment {
 .globalNav .colorize.item-home:hover {
     color: #32325d
 }
+// why-nighthawk-car-section
+.section-cust-head{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+.card-wrapper{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+    
+    margin: 2rem 0rem;
+    padding: 2rem 0rem;
+    .card-item{
+        text-align: center;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        border-radius:5px;
+        margin:0rem 1rem;
+        height: 22rem;
+        box-shadow:0 2px 10px rgba(0,0,0,0.4);

```

Comment:
Decrease the `box-shadow` to bare minimum
![chrome_rNGOhrTFG6](https://user-images.githubusercontent.com/54861487/108637529-0fea9f80-74b1-11eb-86a6-384ff9672d58.png)